CMakeLists.txt

```diff
@@ -703,15 +703,26 @@ psibase_package(
 )
 
 psibase_package(
-    OUTPUT ${SERVICE_DIR}/Default.psi
-    NAME Default
+    OUTPUT ${SERVICE_DIR}/DevDefault.psi
+    NAME DevDefault
```

Comment:
`psibase boot` needs to be updated

Reply:
Tester as well, yeah?
https://github.com/gofractally/psibase/blob/98f3198a66f4754b29468ce783bd7a3496fd6219/rust/psibase/src/main.rs#L646
and
https://github.com/gofractally/psibase/blob/98f3198a66f4754b29468ce783bd7a3496fd6219/rust/psibase/src/tester.rs#L62